src/test/java/com/puppycrawl/tools/checkstyle/internal/AllChecksTest.java

```diff
@@ -159,7 +159,8 @@
                 // we prefer no spaces on one side or both for these tokens
                 "ARRAY_INIT",
                 // these are covered by GenericWhitespaceCheck
-                "WILDCARD_TYPE", "GENERIC_END", "GENERIC_START").collect(Collectors.toSet()));
+                "WILDCARD_TYPE", "GENERIC_END", "GENERIC_START",
+                "ELLIPSIS").collect(Collectors.toSet()));
```

Comment:
ELLIPSIS is not "covered by GenericWhitespaceCheck"
please put it above this comment.
commend should be "we prefer no space around '...' the same as '[]'"